timesketch/api/v1/resources.py

```diff
@@ -826,10 +846,18 @@ def post(self, sketch_id):
                 query_filter,
                 query_dsl,
                 indices,
-                aggregations=None,
+                aggregations=index_stats_agg,
                 return_fields=return_fields,
                 enable_scroll=enable_scroll)
 
+        # Get number of matching documents per index.
+        count_per_index = {}
+        try:
+            for bucket in result['aggregations']['indices']['buckets']:
+                count_per_index[bucket['key']] = bucket['doc_count']
```

Comment:
ok, so the way this works now is that if a single key doesn't exist a KeyError will be raised, can happen in several different places here, and then nothing more will be stored.... what if you actually get back the `result['aggregations']['indices']['buckets']`, so you'll get some `bucket` attributes... and then a single bucket will not have a `key` value (may not happen), but the subsequent ones would... that would mean you don't get all the values you could have gotten... would it be better to make the assignment more error prone?

```
key = bucket.get('key')
if key:
    count_per_index[key] = bucket.get('doc_count')
```

or something like that
count_per_index[

Reply:
Good catch, fixed now.